README.md

```diff
@@ -579,6 +579,12 @@ More .NET Podcasts can be found on [The Sound of .NET](https://thesoundof.net/)
   * **Frequency**: Approximately twice a month
   * **Runtime**: 40 - 80 mins, regularly ~60 mins
 
+* [Eat Sleep Code](https://developer.telerik.com/community/eat-sleep-code/)
```

Comment:
Not in alphabetical order

Reply:
I always think it's Sleep Eat Code for some reason.

It should be good to go now.